compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/expressions/BLangExpression.java

```diff
@@ -51,4 +52,12 @@
     public BType expectedType;
 
     public Map<BVarSymbol, NarrowedTypes> narrowedTypeInfo;
+
+    /**
+     * Indicates whether the expression is an accessible expression.
```

Comment:
Since we are removing the class BLangAccessibleExpression I think we need to this comment to say something like
`Indicates whether the expression is part of a field access expression` or something similar.

Reply:
Fixed in: 8cfc9e7